android/app/src/main/rust/src/runtime.rs

```diff
@@ -43,453 +44,593 @@ use crate::util::{unwrap_exc_or, unwrap_exc_or_default, unwrap_jni};
 use crate::{JniError, JniErrorKind};
 
 const CHAT_SERVICE_EVENTS_HANDLER_FIELD_TYPE: &str =
-    "Lio/locha/p2p/runtime/RuntimeEvents;";
+  "Lio/locha/p2p/runtime/RuntimeEvents;";
+
+pub mod items {
+  include!(concat!(env!("OUT_DIR"), "/message.items.rs"));
+}
+
+#[derive(Serialize, Deserialize)]
+struct Msg {
+  text: String,
+  typeFile: Option<String>,
+  file: Option<String>,
+}
+
+#[derive(Serialize, Deserialize)]
+struct DataStatus {
+  status: String,
+  msgID: String,
+}
+
+#[derive(Serialize, Deserialize)]
+struct Person {
```

Comment:
Person -> Message :wink:

Reply:
Message is already being used and there is an error, rename it MessageData